.circleci/config.yml

```diff
@@ -1,17 +1,81 @@
-version: 2
-jobs:
+version: 2.1
+
+orbs:
+  architect: giantswarm/architect@0.8.14
+
+workflows:
   build:
-    machine: true
-    steps:
-    - checkout
+    jobs:
+      - architect/go-build:
+          name: go-build
+          binary: ingress-exporter
+          filters:
+            tags:
+              only: /^v.*/
+
+      - architect/push-to-docker:
+          context: architect
+          name: push-ingress-exporter-to-quay
+          image: "quay.io/giantswarm/ingress-exporter"
+          username_envar: "QUAY_USERNAME"
+          password_envar: "QUAY_PASSWORD"
+          requires:
+            - go-build
+          filters:
+            # Trigger the job also on git tag.
+            tags:
+              only: /^v.*/
```

Comment:
Shouldn't we push to aliyun as well? Let's fix that here.

Reply:
Historically, this component has never been pushed to Aliyun. Additional info: https://gigantic.slack.com/archives/CHAHY6VMF/p1589283447412400

Reply:
Oh yes, this is only for KVM.